server/router/user.js

```diff
@@ -1,7 +1,10 @@
 const router = require('express').Router();
 
 const { isAuthController } = require('../controller/user');
+const { orderAcceptValidation } = require('../middleware/validation');
+const { acceptOrderController } = require('../controller');
 
 router.get('/is-auth', isAuthController);
+router.post('/provider/orders', orderAcceptValidation, acceptOrderController);
```

Comment:
this route relates to the provider file and needs to providerProtectedMIddelware the user shouldn't can access these endpoints